Package/ExcelDna.AddIn/content/ExcelDna.Build.props

```diff
@@ -0,0 +1,47 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="ExcelDnaProps">
+<!--
+  If you change properties in this file, they may not come into effect until you:
+     * Rebuild the solution/project
+
+     or
+
+     * Close Visual Studio
+     * Delete .vs folder, if exists
+     * Delete ProjectName.csproj.user (or equivalent for VB, F#, etc.), if exists
+     * Delete SolutionName.suo, if exists
+     * Open your solution/project again in Visual Studio
+  -->
+
+  <!--
+    Configuration properties for building .dna files
+  -->
+  <PropertyGroup>
+    <!--
+      Enable/Disable automatic generation of platform-specific versions of .dna files
+    -->
+    <ExcelDnaCreate32BitAddIn Condition="'$(ExcelDnaCreate32BitAddIn)' == ''">true</ExcelDnaCreate32BitAddIn>
+    <ExcelDnaCreate64BitAddIn Condition="'$(ExcelDnaCreate64BitAddIn)' == ''">true</ExcelDnaCreate64BitAddIn>
+
+    <!--
+      Define the suffix used for each platform-specific file e.g. MyAddIn64.dna
+    -->
+    <ExcelDna32BitAddInSuffix Condition="'$(ExcelDna32BitAddInSuffix)' == ''"></ExcelDna32BitAddInSuffix>
+    <ExcelDna64BitAddInSuffix Condition="'$(ExcelDna64BitAddInSuffix)' == ''">64</ExcelDna64BitAddInSuffix>
+  </PropertyGroup>
+
+  <!--
+    Configuration properties for packing .dna files
+  -->
+  <PropertyGroup>
+    <!--
+      Enable/Disable packing of .dna files
+    -->
+    <RunExcelDnaPack Condition="'$(RunExcelDnaPack)' == ''">true</RunExcelDnaPack>
+
+    <!--
+      Suffix used for packed .xll files e.g. MyAddIn-packed.xll
+    -->
+    <ExcelDnaPackXllSuffix Condition="'$(ExcelDnaPackXllSuffix)' == ''">-packed</ExcelDnaPackXllSuffix>
```

Comment:
Good question. I didn't test this. It will do whatever `ExcelDnaPack.exe` decides to do... I'd expect it to override the original XLL, but it's worth testing it. In case it doesn't override, we could add validation to make sure this is never empty.